app/src/main/java/gholzrib/arctouchchallenge/core/utils/PreferencesManager.java

```diff
@@ -0,0 +1,111 @@
+package gholzrib.arctouchchallenge.core.utils;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+import com.google.gson.Gson;
+
+import gholzrib.arctouchchallenge.core.models.TMDBConfiguration;
+
+/**
+ * Created by Gunther Ribak on 01/12/2016.
+ * For more information contact me
+ * through guntherhr@gmail.com
+ */
+public class PreferencesManager {
+
+    private static final String TMDB_CONFIGURATION_KEY = "tmdb_configuration_key";
+    private static final String TMDB_CONFIGURATION_LAST_UPDATE_KEY = "tmdb_configuration_last_update_key";
+
+    private static SharedPreferences getDefaultPreferencesManager(Context context) {
+        return PreferenceManager.getDefaultSharedPreferences(context);
+    }
+
+    private static void setInt(Context context, String key, int value) {
```

Comment:
This class was also created before the challenge. It has all this method's to give the developer support for different data types to be saved.